packages/site-admin/src/components/sidebar/stories/index.stories.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import { __experimentalItemGroup as ItemGroup } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { archive, Icon, navigation, settings } from '@wordpress/icons';
+/**
+ * Internal dependencies
+ */
+import { SidebarContent, SidebarNavigationItem, SidebarNavigationScreen } from '../../';
+import { useLocation, RouterProvider } from '../../../router';
+/**
+ * Types
+ */
+import type { SidebarContentProps } from '..';
+import type { Meta, StoryFn } from '@storybook/react';
+
+import './style.stories.scss';
+
+/**
+ * Storybook metadata
+ */
+const meta: Meta< typeof SidebarContent > = {
+	title: 'Components/SidebarContent',
+	component: SidebarContent,
+};
+
+export default meta;
+
+const SidebarItems = () => {
+	const { path } = useLocation();
+
+	return (
+		<ItemGroup>
+			<SidebarNavigationItem
+				icon={ <Icon icon={ navigation } /> }
```

Comment:
The `SidebarNavigationItem` should accept the icon itself for its icon prop, and not the `<Icon />` component

```suggestion
				icon={ navigation }
```

The same applies to other places where we pass an `<Icon />` component to `SidebarNavigationItem` (including docs)